meetings/meetings/settings/defaults.py

```diff
@@ -180,17 +180,6 @@
     'localhost:4200',
 )
 
-CORS_ALLOW_HEADERS = (
```

Comment:
I had added that because `cache-control` header was causing CORS issues. I can't remember for what request though. Just something to look out for.